src/event_form/steps/event-form-step3.html

```diff
@@ -4,13 +4,13 @@
   <form name="step3Form" class="css-form">
     <h2 class="title-border">
       <span class="number">3</span>
-      <span ng-show="eventFormData.isEvent">Waar vindt dit evenement of deze activiteit plaats?</span>
-      <span ng-show="eventFormData.isPlace">Waar is deze plaats of locatie?</span>
+      <span ng-show="eventFormData.isEvent">{{::translateEventForm('step3', 'title_event')}}</span>
+      <span ng-show="eventFormData.isPlace">{{::translateEventForm('step3', 'title_place')}}</span>
     </h2>
 
     <div class="row">
       <div class="col-xs-12">
-        <label for="gemeente-autocomplete" id="gemeente-label" ng-hide="selectedCity">Kies een gemeente</label>
+        <label for="gemeente-autocomplete" id="gemeente-label" ng-hide="selectedCity">{{::translateEventForm('step3', 'choose_city')}}</label>
         <div id="gemeente-kiezer" ng-hide="selectedCity">
           <span style="position: relative; display: inline-block; direction: ltr;" class="twitter-typeahead">
             <input type="text"
```

Comment:
Placeholders kunnen ook vertaald worden. Het makkelijkst kan dat met de translate filter: `{ 'key.for.translation' | translate }`.

Reply:
Die was ik inderdaad vergeten.
Inmiddels toegevoegd!